From ab8b54cfc9d3fddba297c343a2d8dea9dbe20017 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=C3=B6rg=20Krause?= <joerg.krause@embedded.rocks>
Date: Tue, 4 Aug 2015 22:13:20 +0200
Subject: [PATCH] drop configh from tools
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

We need to build u-boot tools without a board configuration for the target.
fw_env just uses config.h to define the default environment of the created
image, so it really isn't mandatory.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>
[Jörg Krause: update for version 2015.07]
Signed-off-by: Jörg Krause <joerg.krause@embedded.rocks>
[Romain: update for version 2017.05
    after commit 9d80b49a671c9922931adcd823aab0ed319a42d1]
Signed-off-by: Romain Naour <romain.naour@gmail.com>
[Pierre-Jean: update for version 2019.04]
Signed-off-by: Pierre-Jean Texier <pjtexier@koncepto.io>
---
 tools/env/fw_env_private.h | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/tools/env/fw_env_private.h b/tools/env/fw_env_private.h
index 86be16d..3c62ac4 100644
--- a/tools/env/fw_env_private.h
+++ b/tools/env/fw_env_private.h
@@ -4,17 +4,6 @@
  * Wolfgang Denk, DENX Software Engineering, wd@denx.de.
  */
 
-/* Pull in the current config to define the default environment */
-#include <linux/kconfig.h>
-
-#ifndef __ASSEMBLY__
-#define __ASSEMBLY__ /* get only #defines from config.h */
-#include <config.h>
-#undef	__ASSEMBLY__
-#else
-#include <config.h>
-#endif
-
 /*
  * To build the utility with the static configuration
  * comment out the next line.
-- 
2.9.4

